Alignment/APEEstimation/python/conditions/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
@mteroerd could you please clarify the use of this file? I understand that this new directory is empty by default, and filled by the running workflow, is it correct?

Reply:
This file only exists to have the folder, because I did not find a way to add an empty folder to git.

If one specifies certain options in the workflow, files will be generated in the folder and for those files to be found by the importlib in test/cfgTemplate/apeEstimator_cfg.py, I found that the folder needs to have existed before scram was run because apparently at that stage, the folders are added to the search path.

Reply:
@smuzaffar @davidlange6 do we have other similar situations within CMSSW< that you are aware of?

Reply:
@fabiocos , not that I know.
@mteroerd , you can create an empty __init__.py in the directory if you want to keep this directory in git.

Reply:
@smuzaffar Thanks for the answer. I will shortly push a version with an empty init.py to this branch.